src/pages/shop/shop.scss

```diff
@@ -1,4 +1,410 @@
-.shop{
-    position: relative;
-   
+@import '../../vendors/variables';
+@import '../../vendors/responsive.scss';
+
+
+.shop {
+    height: 150vh;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
1. are you sure about your limited height? what will be if we show all the products at one page?
2. don't you have font-family in index.scss?